### PR TITLE
Upgrade to Laravel 8

### PR DESCRIPTION
**Pull request by jfernancordova**
Issue #15

### CODE REVIEW

**Comment by jfernancordova** [comment]
@Devsome Hi!, I did the upgrade from master, because there are a lot of change between develop and master, I think it's better to create a new branch from master for this PR.

**Comment by Devsome** [comment]
Hello @jfernancordova , first of all thanks for your pull-request. I will have a look at it and get back to you as soon as possible.

**Comment by jfernancordova** [comment]
> Hello @jfernancordova , first of all thanks for your pull-request. I will have a look at it and get back to you as soon as possible.

You're welcome!!!

File: database/seeds/DatabaseSeeder.php [hunk 1]
```diff
@@ -1,21 +0,0 @@
-<?php
```
Comment (Devsome) [inline database/seeds/DatabaseSeeder.php:1]: @jfernancordova why are the Seeder all deleted? Any reason for that in Laravel8?

Comment (jfernancordova) [reply database/seeds/DatabaseSeeder.php:1]: I'm sorry, with laravel 8 the folder seeds is changed to a new folder called seeders and my IDE moved them to non-versioned files, that's why I don't push new changes.

**Review by Devsome — COMMENTED** [review]
Upgrading PHP 7.3 to 7.4 atm and then deploying it. After that I will mark this pr with **hacktoberfest-accepted**. 👍

**Comment by jfernancordova** [comment]
> Upgrading PHP 7.3 to 7.4 atm and then deploying it. After that I will mark this pr with **hacktoberfest-accepted**. +1

Perfect, let me know anything that I can fix o change, thx.

**Review by Devsome — APPROVED** [review]
I checked out the pr and pushed it, but dunno how - they created a new branch. Needed to updated the Seeder with that use
```php
use Illuminate\Support\Facades\DB;
```

Updated Vagrant and bootstrap.sh for Vagrant. Everything is working fine now

**Comment by jfernancordova** [comment]
> I checked out the pr and pushed it, but dunno how - they created a new branch. Needed to updated the Seeder with that use
> 
> ```
> use Illuminate\Support\Facades\DB;
> ```
> 
> Updated Vagrant and bootstrap.sh for Vagrant. Everything is working fine now

Excellent @Devsome, thank you!

**Pull request merged**